code/modules/mob/living/simple_animal/hostile/terror_spiders/black.dm

```diff
@@ -0,0 +1,39 @@
+
+// --------------------------------------------------------------------------------
+// ----------------- TERROR SPIDERS: T2 BLACK TERROR ------------------------------
+// --------------------------------------------------------------------------------
+// -------------: ROLE: assassin, poisoner, DoT expert
+// -------------: AI: attacks to inject its venom, then retreats. Will inject its enemies multiple times then hang back to ensure they die.
+// -------------: SPECIAL: venom that does more damage the more of it is in you
+// -------------: TO FIGHT IT: if bitten once, retreat, get charcoal/etc treatment, and come back with a gun.
+// -------------: CONCEPT: http://tvtropes.org/pmwiki/pmwiki.php/Main/GradualGrinder
```

Comment:
Why not? They're harmless design references which illustrate the thinking behind what the mob is meant to do.